### PR TITLE
Hide Products BN and Connect + Sort LIst Products

### PR DESCRIPTION
**Pull request by Rajandeep98**
Hide Products BN and Connect + Sort LIst Products

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://developer-connect-dev--pr-165-q8tqspgl.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://developer-connect-dev--pr-165-q8tqspgl.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://developer-connect-dev--pr-165-q8tqspgl.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://developer-connect-dev--pr-165-sd3eot4i.web.app

File: web/site/app/pages/products/index.vue [hunk 1]
```diff
@@ -16,7 +16,9 @@ const { data: products } = await useAsyncData(
     watch: [locale]
   }
 )
-
+const completedProducts = computed(() => {
+  return products.value?.sort((a, b) => a.name.localeCompare(b.name)) ?? []
+})
 // console.log(products.value)
```
Comment (sameer0422) [inline web/site/app/pages/products/index.vue:22]: Please remove console logs

**Review by sameer0422 — APPROVED** [review]
Looks good

**Pull request merged**